core/dronecore.h

```diff
@@ -75,7 +75,7 @@ class DroneCore {
      * To accept only local connections of the machine, use 127.0.0.1.
      * For any incoming connections, use 0.0.0.0.
      *
-     * @param local_bind_ip The local UDP ip to listen to.
+     * @param local_ip The local UDP ip to listen to.
```

Comment:
The local UDP IP address to listen to. 
OR
The local UDP address to listen to.